src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  filter: Filter;
+  setFilter: (filter: Filter) => void ;
+};
+
+export const TodoFilter: React.FC<Props> = ({ filter, setFilter }) => {
+  const onFilterChange = (status: Filter) => () => {
+    setFilter(status);
+  };
+
+  return (
+    <ul className="filters">
+      <li>
+        <a
+          href="#/"
+          className={filter === 'all' ? 'selected' : ''}
```

Comment:
also, I suggest you can create some links array and render them in the loop(using the .map)